risk_models/strategy.py

```diff
@@ -79,7 +79,7 @@ class BoolStrategy(Strategy):
 
     def __init__(self, d):
         super(BoolStrategy, self).__init__(d)
-        self.func_name = d['strategy_func']
+        self.__name__ = d['strategy_func']
```

Comment:
`func_name` can not be changed here, it's not a build in attribute name but a custom attribute.